packages/aws-cdk-lib/aws-rds/lib/cluster-engine.ts

```diff
@@ -385,11 +390,21 @@ export class AuroraMysqlEngineVersion {
    * @deprecated Version 5.7.mysql_aurora.2.04.8 is no longer supported by Amazon RDS.
    */
   public static readonly VER_2_04_8 = AuroraMysqlEngineVersion.builtIn_5_7('2.04.8');
+  /**
+  * Version "5.7.mysql_aurora.2.04.9"
+  * @deprecated Version 5.7.mysql_aurora.2.04.9 is no longer supported by Amazon RDS.
+  */
+  public static readonly VER_2_04_9 = AuroraMysqlEngineVersion.builtIn_5_7('2.04.9');
   /**
    * Version "5.7.mysql_aurora.2.05.0".
    * @deprecated Version 5.7.mysql_aurora.2.05.0 is no longer supported by Amazon RDS.
    */
   public static readonly VER_2_05_0 = AuroraMysqlEngineVersion.builtIn_5_7('2.05.0');
+  /**
+  * Version "5.7.mysql_aurora.2.05.1"
+  * @deprecated Version 5.7.mysql_aurora.2.05.1 is no longer supported by Amazon RDS.
+  */
+  public static readonly VER_2_05_1 = AuroraMysqlEngineVersion.builtIn_5_7('2.05.1');
```

Comment:
Cannot find this version in the [release notes](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraMySQLReleaseNotes/AuroraMySQL.Updates.20Updates.html).

Reply:
Again, based on the SDK, it's not listed. It could have been removed, or never existed:

```sh
$ aws rds describe-db-engine-versions --engine aurora-mysql --region us-east-1 | jq -r '.DBEngineVersions[] | select(.EngineVersion | endswith("2.05.1"))'
```